ui/src/hooks/useFileManage.ts

```diff
@@ -551,61 +537,45 @@ export const useFileManage = (fileConnectionUrl: string) => {
           onError: () => void;
           onProgress: (e: { percent: number }) => void;
         }) => {
-          if (socket) {
-            handleFileUpload(socket, fileList, onProgress, onFinish, onError);
-          }
+          handleFileUpload(<WebSocket>socket, fileList, onProgress, onFinish, onError);
         }
       );
 
       mittBus.on('download-file', ({ path, is_dir }: { path: string; is_dir: boolean }) => {
-        if (socket) {
-          handleFileDownload(socket, path, is_dir);
-        }
+        handleFileDownload(<WebSocket>socket, path, is_dir);
       });
 
       mittBus.on(
         'file-manage',
         ({ path, type, new_name }: { path: string; type: ManageTypes; new_name?: string }) => {
-          if (!socket) return;
-
           switch (type) {
             case ManageTypes.CREATE: {
-              handleFileCreate(socket, path);
+              handleFileCreate(<WebSocket>socket, path);
               break;
             }
             case ManageTypes.CHANGE: {
-              handleChangePath(socket, path);
+              handleChangePath(<WebSocket>socket, path);
               break;
             }
             case ManageTypes.REFRESH: {
-              refresh(socket, path);
+              refresh(<WebSocket>socket, path);
               break;
             }
             case ManageTypes.RENAME: {
-              handleFileRename(socket, path, new_name!);
+              handleFileRename(<WebSocket>socket, path, new_name!);
               break;
             }
             case ManageTypes.REMOVE: {
-              handleFileRemove(socket, path);
+              handleFileRemove(<WebSocket>socket, path);
               break;
             }
           }
         }
       );
-
-      return {
-        cleanup,
-        socket
-      };
-    } catch (error) {
-      cleanup();
-      console.error('File management initialization failed:', error);
-      
-      throw error;
     }
   }
 
-  return init();
+  init();
 };
 
 export const unloadListeners = () => {
```

Comment:
As there's no specific input or requirement in the question for checking code diffs, potential issues, or optimization suggestions. However, generally speaking, this looks like a simple React component setup with some WebSocket and event handling logic that you need to check.

It seems fine but always good practice would be looking into how data flows through each step of the process to ensure it's clear and efficient.

For any more specific queries, I'd recommend providing detailed information about what you want checked, which part you're concerned with, etc.